Content.Client/Physics/Controllers/MoverController.cs

```diff
@@ -52,21 +53,11 @@ public override void UpdateBeforeSolve(bool prediction, float frameTime)
             if (_playerManager.LocalPlayer?.ControlledEntity is not {Valid: true} player)
                 return;
 
-            if (TryComp<RelayInputMoverComponent>(player, out var relayMover))
+            if (TryComp<RelayInputMoverComponent>(player, out var relayMover)
+                && TryComp(relayMover.RelayEntity, out MovementRelayTargetComponent? targetComp))
             {
-                if (relayMover.RelayEntity != null)
-                {
-                    if (TryComp<InputMoverComponent>(player, out var mover) &&
-                        TryComp<InputMoverComponent>(relayMover.RelayEntity, out var relayed))
-                    {
-                        relayed.CanMove = mover.CanMove;
-                        relayed.RelativeEntity = mover.RelativeEntity;
-                        relayed.RelativeRotation = mover.RelativeRotation;
-                        relayed.TargetRelativeRotation = mover.RelativeRotation;
-                    }
-
-                    HandleClientsideMovement(relayMover.RelayEntity.Value, frameTime);
-                }
+                DebugTools.Assert(targetComp.Entities.Count <= 1, "Multiple relayed movers are not supported at the moment");
```

Comment:
I just sprinkled these in a few places that need to be updated before multiple movers is supported.